CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+- Populate `users` field of the public Shift GET API with `rolling_users` from the type override created from web UI([#3303](https://github.com/grafana/oncall/pull/3303))
```

Comment:
It looks like there is one [minor last change needed](https://github.com/grafana/oncall/actions/runs/6881408609/job/18717982505?pr=3303) to make lint happy. After that this should be ready to merge :+1: